qrenderdoc/Code/Interface/RemoteHost.h

```diff
@@ -49,7 +49,7 @@ class RemoteHost
       "Ping the host to check current status - if the server is running, connection status, etc.");
   void CheckStatus();
   DOCUMENT("Runs the command specified in :data:`runCommand`.");
```

Comment:
You need to update the `DOCUMENT` here to mention the return value.

Reply:
I have added the return value to the `DOCUMENT`.